config/karma-test-shim.js

```diff
@@ -1,22 +1,42 @@
 Error.stackTraceLimit = Infinity;
```

Comment:
This might be dangerous when there is infinite recursion... default value is 10, something like 30 or 50 is usually sufficient.